src/components/PortfolioCard/CardTablet.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { FC } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Card, CardActionArea, CardContent, CardMedia, Typography } from '@material-ui/core';
+import { PhoneIphone } from '@material-ui/icons';
```

Comment:
In the project, we're using feather icons

Reply:
Alrighty, will change as suggested